src/up.cmd.js

```diff
@@ -143,11 +131,64 @@ export default class UpCommand extends AbstractCommand {
 
     try {
       await this._project.init();
+
+      this.watchGit();
```

Comment:
I think this only needs to be done, if no explicit `--url` is set via arguments.